### DISCUSSION TITLE
[Help]: Can we use breakpoints to use Fade transitions?

### DISCUSSION THREAD

**Discussion by psntr** [davidjerleke/embla-carousel#975] [Help] [answered]
### Summary

Hi, 

as the title suggests, I wonder if it's possible to swap to `Fade` transition when reaching a certain `breakpoints`? Or I should manually do it with `destroy()` and create a new upon `window.resize `event?

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment, marked as answer]
@psntr do you mean that you would like to activate the Fade plugin on a certain breakpoint? If yes, use the breakpoints option for the Fade plugin like so:

```tsx
const emblaNode = document.querySelector('.embla')
const emblaApi = EmblaCarousel(emblaNode, { loop: true }, [Fade({
  breakpoints: {
    '(min-width: 768px)': { active: false }
  }
})])
```

**Reply by psntr** [reply]
Hi @davidjerleke, yes exactly!

Awesome! I didn't know you could pass the breakpoints option inside the `Fade` plugin! 

Thank you so much

**Reply by davidjerleke** [reply]
@psntr glad I could help. All [official plugins](https://www.embla-carousel.com/plugins/) support the `breakpoints` and `active` option.
